docs/API/fliplet-ui-actions.md

```diff
@@ -29,7 +29,9 @@ Fliplet.UI.Actions(options)
   * **title** (String) A title that appears above the options.
   * **labels** (Array) Options to show the user. Each label object contains the following properties:
     * **label** (String) Option label to show the user.
-    * **action** (Object or Function) If an object is provided the object will be passed to `Fliplet.Navigate.to()` and executed accordingly. If a function is provided, the function will be run with the 0-based index of the label as the first parameter.
+    * **action** (Object or Function)
+      * (Object) If the object contains a `type` key with a value of `copyText`, the value for `text` will be copied ot the clipboard. Otherwisem the object will be passed to `Fliplet.Navigate.to()` and executed accordingly.
```

Comment:
```suggestion
      * (Object) If the object contains a `type` key with a value of `copyText`, the value for `text` will be copied ot the clipboard. Otherwise, the object will be passed to `Fliplet.Navigate.to()` and executed accordingly.
```